localisation/wc_culture_buildings.csv

```diff
@@ -1,97 +1,743 @@
+wc_tuskarr_warcamp1_desc;Tuskarr Mustering Grounds;;;;;;;;;;;;;x
+wc_tuskarr_warcamp2_desc;War-tusk Camp;;;;;;;;;;;;;x
+wc_tuskarr_warcamp3_desc;Kalu'ak War-grounds;;;;;;;;;;;;;x
+wc_tuskarr_warcamp4_desc;Kalu'ak Assembly;;;;;;;;;;;;;x
+wc_tuskarr_tribalship1_desc;Turtle Breeding Grounds;;;;;;;;;;;;;x
+wc_tuskarr_tribalship2_desc;Turtle Docks;;;;;;;;;;;;;x
+wc_tuskarr_tribalship3_desc;Turtle Wardocks;;;;;;;;;;;;;x
+wc_tuskarr_tribalship4_desc;Turtle Rider Drydocks;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+#Unique Human Cultures;;;;;;;;;;;;;;x
+wc_azer_barracks1;7th Legion Encampment;;;;;;;;;;;;;x
+wc_azer_barracks2;7th Legion Barracks;;;;;;;;;;;;;x
+wc_azer_barracks3;7th Legion War Keep;;;;;;;;;;;;;x
+wc_azer_barracks4;7th Legion District;;;;;;;;;;;;;x
+wc_azer_mageguild1;Mage Quarter;;;;;;;;;;;;;x
+wc_azer_mageguild2;Mage Tower;;;;;;;;;;;;;x
+wc_azer_mageguild3;Mage Guild;;;;;;;;;;;;;x
+wc_azer_mageguild4;Magic Keep;;;;;;;;;;;;;x
+wc_azer_militia1;Militia Holdout;;;;;;;;;;;;;x
+wc_azer_militia2;Militia Hall;;;;;;;;;;;;;x
+wc_azer_militia3;Militia Tower;;;;;;;;;;;;;x
+wc_azer_militia4;Militia Fortified Tower;;;;;;;;;;;;;x
+wc_azer_wharf1;Lighthouse;;;;;;;;;;;;;x
+wc_azer_wharf2;Naval Garrison;;;;;;;;;;;;;x
+wc_azer_wharf3;Royal Dockyard;;;;;;;;;;;;;x
+wc_azer_wharf4;Royal Marina;;;;;;;;;;;;;x
+wc_azer_macademy1;Bishop Offices;;;;;;;;;;;;;x
+wc_azer_macademy2;Bishop Library;;;;;;;;;;;;;x
+wc_azer_macademy3;Bishop Mission;;;;;;;;;;;;;x
+wc_azer_macademy4;Bishop Palace;;;;;;;;;;;;;x
+wc_azer_legionary1;Legionary Encampment;;;;;;;;;;;;;x
+wc_azer_legionary2;Legionary Tent City;;;;;;;;;;;;;x
+wc_azer_legionary3;Legionary Fort;;;;;;;;;;;;;x
+wc_azer_legionary4;Legionary Barracks;;;;;;;;;;;;;x
+wc_azer_noblecamp1;Noble's Hut;;;;;;;;;;;;;x
+wc_azer_noblecamp2;Noble Fortified Hut;;;;;;;;;;;;;x
+wc_azer_noblecamp3;Noble Manor;;;;;;;;;;;;;x
+wc_azer_noblecamp4;Noble Estate;;;;;;;;;;;;;x
+wc_azer_noblecouncil1;Noble Embassy;;;;;;;;;;;;;x
+wc_azer_noblecouncil2;Noble Council;;;;;;;;;;;;;x
+wc_azer_noblecouncil3;Noble Chateau;;;;;;;;;;;;;x
+wc_azer_noblecouncil4;Noble Summer Palace;;;;;;;;;;;;;x
+wc_azer_barracks1_desc;The 7th Legion is an elite armed forces composed primarily of Alliance veterans. This encampment helps train and recruit new members.;;;;;;;;;;;;;x
+wc_azer_barracks2_desc;The 7th Legion is an elite armed forces composed primarily of Alliance veterans. A barracks where the 7th Legion is stationed and trains.;;;;;;;;;;;;;x
+wc_azer_barracks3_desc;The 7th Legion is an elite armed forces composed primarily of Alliance veterans. A specialized keep not made for nobles, but for commanders and officers made to be a forward base on possible battlefronts.;;;;;;;;;;;;;x
+wc_azer_barracks4_desc;The 7th Legion is an elite armed forces composed primarily of Alliance veterans. A veteran's district and informal pension where legionary families can train new generations for the 7th Legion in relative military luxury.;;;;;;;;;;;;;x
```

Comment:
Can Alliance veterans be recruited before creation of the Alliance?
Can Alliance veterans fight on the Horde side against Alliance?
:thonk: